src/components/professor/databases/CreateDatabase.vue

```diff
@@ -44,28 +45,24 @@ export default {
   },
 
   methods: {
-    createDatabase () {
-      // NOTE Check whether database with same name already exists.
-      axios.post('http://localhost:3000/databases',{
-        name: this.database.name,
-        description: this.database.description,
-        url_diagram: this.database.url_diagram
-      })
-      .then(({ data: database }) => {
-        this.$store.commit('databases/addDatabase', database)
-
-        // Clear input data
-        this.database.name = 'Database'
-        this.database.description = 'Description'
-        this.database.url_diagram = 'URL Diagram'
+    ...mapActions({
+      newDatabase: 'databases/createDatabase'
+    }),
 
-        this.$router.push({ name: 'Database', params: { id: database.id }})
-        toastr.success('Creating database', 'Successful')
+    createDatabase () {
+      this.newDatabase(this.database)
+        .then(({ id }) => {
```

Comment:
Make sure this `.then` and the `.catch` below are indented the same. Not sure if your linter should pick up on this?